### DISCUSSION TITLE
Want to republish VedicDateTime on CRAN

### DISCUSSION THREAD

**Discussion by prajwalkpatil** [rbind/yihui#1475] [Q&A] [answered]
VedicDateTime was removed from CRAN - [CRAN Link](https://cran.r-project.org/web/packages/VedicDateTime/index.html), Xie had made a Pull Request - [PR link](https://github.com/prajwalkpatil/VedicDateTime/pull/10) to the repository, It was merged but, we didn't release it on CRAN at that time.

Now that the package is removed, we want to fix this and republish it. Can you please tell me what can be done to fix this?

**Comment by yihui** [comment, marked as answer]
Sorry, I should have reminded you of making a new CRAN release, although CRAN must have reminded you. Anyway, to republish the package, the process is the same as publishing any new version of a package, but just explain (with apologies) what happened and what you did in the "Optional comments" field in the submission form. Good luck! Feel free to let me know if you encounter any problem in the submission.
